spec/controllers/categories_controller_spec.rb

```diff
@@ -48,7 +52,8 @@
         create_11_categories
```

Comment:
This test and tests below appear to be describing what your code is doing rather than what the expected behavior is. Try to avoid describing exactly what the code is doing and focus on the behaviors.